qhull.yaml

```diff
@@ -0,0 +1,51 @@
+# Generated from https://git.alpinelinux.org/aports/plain/community/qhull/APKBUILD
+package:
+  name: qhull
+  version: "2020.2"
+  epoch: 0
+  description: Calculate convex hulls and related structures
+  copyright:
+    - license: custom
```

Comment:
I’m guessing this isn’t valid SPDX

Reply:
https://github.com/qhull/qhull/issues/40

I'll look into it